appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -456,6 +490,17 @@
                         (cdr event-info)))
                      events))
 
+         ;; Register the multi events with the Simple event dispatcher
+         (define (register-multi-events events)
+           (define-alias SimpleEventDispatcher
+             <com.google.appinventor.components.runtime.EventDispatcher>)
+           (for-each (lambda (eventName)
+                       ;; Tell the Simple event dispatcher to delegate dispatching of this event to this class
+                       (SimpleEventDispatcher:registerMultiEventForDelegation
+                        (as com.google.appinventor.components.runtime.HandlesEventDispatching (this))
+                        eventName))
```

Comment:
Sorry for the delay. Basically I have multi-events registered separately, and whenever a normal event is called, the correct multi-event will also be dispatched if it exists (currently hardcoded only for buttons right now). Currently, an app using the multi-event block fails when the event is dispatched. When an app using this block is built and you click on a button, the app will crash and it will simply says "Unfortunately APPNAME has stopped." If in the EventDispatcher, if I comment out the line "dispatchDelegate.dispatchMultiEvent(multiEventName, newArgs);" then no error occurs. Similarly, if in the function dispatchMultiEvent in runtime.scm, if I just change it so that it simply returns without doing anything, no error occurs. So this leads to believe that it is somewhere in this snippet of code that everything breaks, but I am not sure what yet.